src/terminal.go

```diff
@@ -1837,8 +1843,13 @@ func (t *Terminal) Loop() {
 		intChan := make(chan os.Signal, 1)
 		signal.Notify(intChan, os.Interrupt, syscall.SIGTERM)
 		go func() {
-			<-intChan
-			t.reqBox.Set(reqQuit, nil)
+			for {
+				s := <-intChan
```

Comment:
We can simplify this to `for s := range intChan {`.

* https://gobyexample.com/range-over-channels